src/components/backend-ai-storage-list.ts

```diff
@@ -1781,6 +1781,17 @@ export default class BackendAiStorageList extends BackendAIPage {
     }
   }
 
+    /**
+   * Hide a wl dialog.
+   *
+   * @param {Event} e - Dispatches from the native input event each time the input changes.
+   */
+  _hideDialog(e) {
+    let hideButton = e.target;
+    let dialog = hideButton.closest('wl-dialog');
+    dialog.hide();
+  }
+
```

Comment:
Introduction `_hideDialog` prevents closing delete file dialog with Cancel button.

Reply:
I just found out that _hideDialog is already implemented in the base class of BackendAiStorageList, which is BackendAIPage.
https://github.com/lablup/backend.ai-console/blob/0db0d2174f051193116e5fd3cc04c61f82f8c8ad/src/components/backend-ai-page.ts#L77-L85

Therefore, I'll remove `_hideDialog` function rightaway.